### DISCUSSION TITLE
Wcjs loads, is able to read vlc version number, but doesn't play files. DebugView says 'libvlc can't find plugins cache file'

### DISCUSSION THREAD

**Discussion by RuurdBijlsma** [RSATom/WebChimera.js#136] [Q&A] [answered]
I use wcjs-prebuilt, which loads wcjs without errors, and the returned object seems fine as it contains the proper VLC version number. However when I create a player and play a MRL nothing happens. No properties on the player change, no sound plays and no message or error events fire. 
When I look at DebugView which I saw linked in another issue related to wcjs I can see that VLC seems to have trouble loading plugins. It tries loading a plugins.dat file (in node-modules/wcjs-prebuilt/bin/plugins/), however this file doesn't exist.

Is there anything I can do to fix this? Is this an issue related to Webchimera.js or something else? I've attached my LOG file from DebugView and a screenshot in case that could be helpful

[DESKTOP-U4ISCOI.LOG](https://github.com/RSATom/WebChimera.js/files/5863043/DESKTOP-U4ISCOI.LOG)
![image](https://user-images.githubusercontent.com/5478546/105641873-9007f000-5e86-11eb-960c-25b4fd5d3ff1.png)

**Comment by RSATom** [comment, marked as answer]
Possibly related: https://github.com/RSATom/WebChimera.js/wiki/Latest-Electron-and-NW.js-versions-has-compatibility-issue-on-Windows
